### DISCUSSION TITLE
Use ctrl+mouse scroll

### DISCUSSION THREAD

**Discussion by wilsonk** [david-vanderson/dvui#16] [Q&A] [answered]
Just wondering how to check for ctrl being held down and also the mouse scroll being used in the sdl-test example? I can capture the ctrl in SDL, but I can't seem to check for it and the '.wheel_y' use at line 60 of sdl-test.zig.  I was just going to change that code to test it out, as Linux and most browsers use the ctrl-mouseWheel for scaling.

**Comment by david-vanderson** [comment]
Great question!  As I understand it, the best way is to separately track the state of the keys so that the application knows during processing the wheel event.

I'm working on it, but it's made me realize that EventIterator was the wrong approach, so I'm reworking that.  I'll comment again once I get something working.

**Comment by david-vanderson** [comment, marked as answer]
Ok that turned out better than I expected.  Take a look at how sdl-test is now processing the events.  It can now watch for key events (but not process them) to record the pressed state of the modifier keys.

Does that look good?

**Reply by wilsonk** [reply]
Yes, that looks good. I have a couple other test changes that I will add over top of these changes and see how they work out. Good stuff.
